src/pages/ProfilePage/views/SellHouseView/AddHouseForm.module.css

```diff
@@ -0,0 +1,55 @@
+.formContainer {
```

Comment:
It looks like this file is duplicated: `src/components/ProfilePage/AddHouseForm/AddHouseForm.module.css`.